src/Microsoft.Framework.Logging.Serilog/SerilogLogger.cs

```diff
@@ -89,6 +89,8 @@ private LogEventLevel ConvertLevel(LogLevel logLevel)
                 case LogLevel.Information:
                     return LogEventLevel.Information;
                 case LogLevel.Verbose:
+                    return LogEventLevel.Debug;
+                case LogLevel.Debug:
                     return LogEventLevel.Verbose;
                 default:
                     throw new NotSupportedException();
```

Comment:
I looked at the log levels in 9 popular loggers, and serilog is the only logger that has debug as a lower level than verbose.  I am open to suggestions for as to what to do in this case